goodfood/complain/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+# Create your models here.
+
+class Complain(models.Model):
+    #author = models.ForeignKey('auth.User', on_delete=models.CASCADE)
+    name = models.CharField(max_length=200)
+    email = models.EmailField( max_length=100, unique=True)
+    university_name = models.CharField(max_length=200, default='')
+    university_address = models.TextField(max_length=400)
+    quality_choice = [('1','bad'),('2','bearable'),('3','meidum'),('4','good'),('5','select')]
```

Comment:
Put it at line 6; in the starting

Reply:
see https://github.com/coala/community/blob/9261c9f6d0777a150e3ee69b146e4d7f52a4eb82/gci/models.py#L55

Reply:
If you wanted, I will change it to `intergerfield` 

>Put it at line 6; in the starting
Nope It does't matter

Reply:
>It does't matter

I know :)
But taht way it looks good and symmetrical.